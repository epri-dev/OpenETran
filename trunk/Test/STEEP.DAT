1 3 100.0 1 1 0.1e-6 400.0e-6

cable 1 2.0 1.5e8 0.0e3

steepfront 30.0e3 3.67e-6 97.7e-6 0.0e-6 2.86
pairs 1 0
poles 2

meter
pairs 1 0
poles 2


