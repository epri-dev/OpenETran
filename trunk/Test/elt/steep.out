  N   span     dT   Tmax
  1 100.00 1e-007 0.0004
Z-phase
  2.00000e+000
Z-modal
  2.00000e+000
Modal Transformation (Ti)
  1.00000e+000
Power-frequency source at pole 1.
   #        vp        vm         i
   1       0.0       0.0       0.0
Power-frequency source at pole 3.
   #        vp        vm         i
   1       0.0       0.0       0.0
Meter at pole 2, from 1 to 0, max V   = 2.995361e+004
nr_iter = 0, nr_max = 0
Execution time [sec]: 0.031
